js/build-directory.js

```diff
@@ -1371,14 +1371,12 @@ DataDirectory.prototype.flViewportRedraw = function() {
       transform: 'scale(1)',
       position: 'absolute'
     });
+    $(document.body).css({
```

Comment:
is this correct? You're changing the css of the same properties a few lines above.

Reply:
Yeh. I'm changing it and the resetting it so that it forces a redraw from the rendering engine.